kinto/plugins/admin/release_hook.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
```

Comment:
A quick module-level docstring linking to the zest.releaser docs and explaining how to test changes to this would be nice.